Assets/FullSerializer/Source/fsConfig.cs

```diff
@@ -53,7 +53,12 @@ public class fsConfig {
         public Func<string, MemberInfo, string> GetJsonNameFromMemberName = (name, info) => name;
 
         /// <summary>
-        /// Should the default serialization behaviour include non-auto properties?
+        /// Should the default serialization behaviour include any properties?
+        /// </summary>
+        public bool SerializeProperties = true;
```

Comment:
Actually I just had a look, and the two existing options (allow non-auto properties, and allow private setter properties) are orthogonal so there would have to be combinations in an enum, e.g.

    enum PropertySerializationConfig {
        None,
        All, // with public or private setter, auto or not
        AutoWithPublicSetter, // only auto, and only if setter is public
        AllAuto, // only auto properties, whether setter is public or not
        AllWithPublicSetter // auto and non-auto properties, but only if the setter is public
    }

If one of those concepts is removed from the enum there will still have to be a bool to control it like there is currently. I think it's more confusing to have an enum and a bool rather than three bools. And even more confusing to have an enum with combinations. But of course it's your call! What do you think?